src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const cost = 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+
+  if (days < 3) {
+    return days * cost;
+  }
+
+  if (days < 7) {
+    return (days * cost) - smallDiscount;
+  }
+
+  return (days * cost) - bigDiscount;
```

Comment:
Make sense to make variable for this operation `(days * cost)`, to prevent code duplicating